libtiledbvcf/src/dataset/attribute_buffer_set.cc

```diff
@@ -25,43 +25,49 @@ void AttributeBufferSet::allocate_fixed(
   nbytes = mem_budget_mb == 0 ? 1024 : (mem_budget_mb * 1024 * 1024);
   num_offsets = nbytes / sizeof(uint64_t);
 
-  using names = TileDBVCFDataset::AttrNames;
+  using attrNames = TileDBVCFDataset::AttrNames;
```

Comment:
`attr_names`, `dim_names` for consistency.

Reply:
There is a parameter passed in to this function that is `attr_names` so we can't use that for the namespace alias. The alternative is to remove the alias and just reference the full namespace in the if statements below.